docs/variants/asus_kgpe_d16/spi-wp.md

```diff
@@ -152,46 +152,46 @@ the chip (running from the tagret machine to be protected):
-	start=0x00200000 length=0x00600000 (upper 3/4)
-	start=0x00000000 length=0x00700000 (lower 7/8)
-	start=0x00100000 length=0x00700000 (upper 7/8)
-	start=0x00000000 length=0x00780000 (lower 15/16)
-	start=0x00080000 length=0x00780000 (upper 15/16)
-	start=0x00000000 length=0x007c0000 (lower 31/32)
-	start=0x00040000 length=0x007c0000 (upper 31/32)
-	start=0x00000000 length=0x007e0000 (lower 63/64)
-	start=0x00020000 length=0x007e0000 (upper 63/64)
-	start=0x00000000 length=0x007f8000 (lower 255/256)
-	start=0x00008000 length=0x007f8000 (upper 255/256)
-	start=0x00000000 length=0x007fc000 (lower 511/512)
-	start=0x00004000 length=0x007fc000 (upper 511/512)
-	start=0x00000000 length=0x007fe000 (lower 1023/1024)
-	start=0x00002000 length=0x007fe000 (upper 1023/1024)
-	start=0x00000000 length=0x007ff000 (lower 2047/2048)
-	start=0x00001000 length=0x007ff000 (upper 2047/2048)
-	start=0x00000000 length=0x00800000 (all)
+ start=0x00000000 length=0x00000000 (none)
+ start=0x00000000 length=0x00001000 (lower 1/2048)
+ start=0x007ff000 length=0x00001000 (upper 1/2048)
+ start=0x00000000 length=0x00002000 (lower 1/1024)
+ start=0x007fe000 length=0x00002000 (upper 1/1024)
+ start=0x00000000 length=0x00004000 (lower 1/512)
+ start=0x007fc000 length=0x00004000 (upper 1/512)
+ start=0x00000000 length=0x00008000 (lower 1/256)
+ start=0x007f8000 length=0x00008000 (upper 1/256)
+ start=0x00000000 length=0x00020000 (lower 1/64)
+ start=0x007e0000 length=0x00020000 (upper 1/64)
+ start=0x00000000 length=0x00040000 (lower 1/32)
+ start=0x007c0000 length=0x00040000 (upper 1/32)
+ start=0x00000000 length=0x00080000 (lower 1/16)
+ start=0x00780000 length=0x00080000 (upper 1/16)
+ start=0x00000000 length=0x00100000 (lower 1/8)
+ start=0x00700000 length=0x00100000 (upper 1/8)
+ start=0x00000000 length=0x00200000 (lower 1/4)
+ start=0x00600000 length=0x00200000 (upper 1/4)
+ start=0x00000000 length=0x00400000 (lower 1/2)
+ start=0x00400000 length=0x00400000 (upper 1/2)
+ start=0x00000000 length=0x00600000 (lower 3/4)
+ start=0x00200000 length=0x00600000 (upper 3/4)
+ start=0x00000000 length=0x00700000 (lower 7/8)
+ start=0x00100000 length=0x00700000 (upper 7/8)
+ start=0x00000000 length=0x00780000 (lower 15/16)
+ start=0x00080000 length=0x00780000 (upper 15/16)
+ start=0x00000000 length=0x007c0000 (lower 31/32)
+ start=0x00040000 length=0x007c0000 (upper 31/32)
+ start=0x00000000 length=0x007e0000 (lower 63/64)
+ start=0x00020000 length=0x007e0000 (upper 63/64)
+ start=0x00000000 length=0x007f8000 (lower 255/256)
+ start=0x00008000 length=0x007f8000 (upper 255/256)
+ start=0x00000000 length=0x007fc000 (lower 511/512)
+ start=0x00004000 length=0x007fc000 (upper 511/512)
+ start=0x00000000 length=0x007fe000 (lower 1023/1024)
+ start=0x00002000 length=0x007fe000 (upper 1023/1024)
+ start=0x00000000 length=0x007ff000 (lower 2047/2048)
+ start=0x00001000 length=0x007ff000 (upper 2047/2048)
+ start=0x00000000 length=0x00800000 (all)
```

Comment:
@macpijan I'm not sure about such changes, can we disable automatic fixes for code blocks?

Reply:
@pietrushnic Yes, actually I have already disabled it here: https://github.com/Dasharo/docs/pull/85/files#diff-39aecbc26791dd03b9a4fdc1f856769ea73308294b7111d70f279ac9bdef35bdR53

Just need to rerun and force push

Reply:
disabled for code blocks